documentation/adr/0002-wiki-platform.md

```diff
@@ -0,0 +1,294 @@
+# Communications Tooling: Wiki Platform
+
+- **Status:** Draft
+- **Last Modified:** 2023-06-15 <!-- REQUIRED -->
+- **Related Issue:** [#{issue number}](https://github.com/HHS/grants-api/issues) <!-- RECOMMENDED -->
+- **Deciders:** {list everyone involved in the decision} <!-- OPTIONAL -->
+- **Tags:** {space and/or comma separated list of tags} <!-- OPTIONAL -->
+
+## Context and Problem Statement
+
+The [communications platform milestone](milestone) identifies a series of platforms through which the Grants API project needs to engage both internal and external stakeholders. One of these platforms is a wiki for storing notes, documents, and other content about the project. Ideally we would select a platform that balances ease of use and flexibility with the cost of implementing and maintaining the wiki.
+
+The goal of this ADR is to evaluate a series of potential wiki platforms and determine which one best fits the needs and objectives of this project based on the decision criteria outlined below.
+
+## Decision Drivers <!-- RECOMMENDED -->
+
+#### Must Have
+
+- **Usability:** Non-technical users should be able to access and create content with minimal training or guidance.
+- **Content Review:** Collaborators should be able to review and edit draft content before those changes are published.
+- **Version History:** Editors should be able to see and restore previous versions of a given page.
+- **Multi-Media:** The platform should support multiple types of media (e.g. videos, images, file uploads, tables, diagrams) with minimal configuration.
+- **Internationalization (i18n):** The platform should provide support for displaying content in multiple languages.
+- **Web Analytics:** The platform should provide support for tracking site usage and other web analytics.
+- **Onboarding Costs:** Onboarding new members to the platform should be relatively inexpensive, both in terms of staff time/resources and direct costs (e.g. licensing fees).
+- **Maintenance Costs:** It should not be prohibitively expensive to maintain the wiki, both in terms of staff time/resources and direct costs (e.g. hosting fees).
+
+#### Nice to Have
```

Comment:
do you want anything about machine readability or structured data here as a nice-to-have? it seems related to 'data access' but not quite the same thing.

Reply:
Yeah that's a really good suggestion. Machine readable and/or structured content feels like it's own goal for sure and definitely a differentiator across wiki platforms

Reply:
This file was moved into #30